public/stylesheets/site/2.0/22-system-messages.css

```diff
@@ -49,6 +49,7 @@ div.error {
   background: #efd1d1;
   border-color: #900;
     box-shadow: 1px 1px 2px;
+    word-wrap: break-word;
```

Comment:
Can we move this up to the style rule for `.notice, .comment_notice, .kudos_notice, ul.notes, .caution, .error, .comment_error, .kudos_error, .alert.flash` instead? I think it makes sense to ensure it will be applied to any of our blue success or yellow caution messages as well.

Reply:
Ah okay you were right about this, it works! Updated.